### DISCUSSION TITLE
Filter drop box does not disappear when blank space clicked

### DISCUSSION THREAD

**Discussion by Leonezz** [community/community#8634] [Other Feature Feedback, Questions, & Ideas] [answered]
![image](https://user-images.githubusercontent.com/33564074/145510084-7c3001e5-c1dd-4dce-b7e7-a5b63d42f9ec.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the heads up! I've added this to our tracking issue for the team to investigate 😊
